databases/week2/databases-week2-Homework/school_databse.sql

```diff
@@ -0,0 +1,35 @@
+SET NAMES utf8mb4;
+
+-- tables
+
+CREATE TABLE `class` (
+`id` int(10) unsigned NOT NULL AUTO_INCREMENT PRIMARY KEY,
+`name` varchar(255) NOT NULL,
+`begins` DATETIME NULL DEFAULT NULL,
+`ends` DATETIME NULL DEFAULT NULL
+)ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci;
+
+CREATE TABLE `student`(
+`id` int(10) unsigned NOT NULL AUTO_INCREMENT PRIMARY KEY,
+`name` varchar(255) NOT NULL,
+`email` varchar(255) NOT NULL,
+`phone` varchar(255) NULL,
+`class_id` int(10) unsigned NOT NULL,
```

Comment:
class_id is not needed when you have the student_classes table.

Reply:
okay